docs/bpmn-support.adoc

```diff
@@ -0,0 +1,122 @@
+= Description of the supported BPMN elements
+:icons: font
+:toc: preamble
+
+NOTE: The support roadmap is available in the link:bpmn-support-roadmap.adoc[BPMN Support Roadmap] page
```

Comment:
It's weird, when I preview the page, **NOTE** is in a array.

Reply:
This is not something new with this PR, it previously existed in other page.
This is something described in https://asciidoctor.org/docs/user-manual/#unicode-admonition-icons

> When the document is processed through the GitHub interface, the shortcodes get replaced with real emojis. This is the only known way to get admonition icons to work on GitHub.

I succeeded make it work for `TIP` with the following, I failed with `NOTE` so I finally decide to not put the configuration. I can eventually retry to make it work, but if so, this should be applied to all doc files.
```
ifdef::env-github[]
:tip-caption: :bulb:
endif::[]
```
Notice that the font-awesome icons I added in the new `bpmn-support.adoc` file cannot be rendered on GitHub